.rubocop.yml

```diff
@@ -1,17 +1,30 @@
-AllCops:
-  TargetRubyVersion: 2.6
+inherit_from: .rubocop_todo.yml
 
-Style/StringLiterals:
-  Enabled: true
-  EnforcedStyle: double_quotes
+inherit_gem:
+  defra_ruby_style:
+    - default.yml
 
-Style/StringLiteralsInInterpolation:
-  Enabled: true
-  EnforcedStyle: double_quotes
+require:
+  - rubocop-factory_bot
+  - rubocop-rake
+  - rubocop-rspec
 
 Layout/LineLength:
   Max: 120
 
 Metrics/BlockLength:
   Exclude:
     - 'spec/**/*'
```

Comment:
These exceptions are already included into default defra-ruby-style config
https://github.com/DEFRA/defra-ruby-style/blob/main/default.yml

This is a brand new gem, so ideally we should not have anything in this config apart from

```
inherit_gem:
  defra_ruby_style:
    - default.yml
```

Reply:
OK, but `Style/OpenStructUse` is problematic so I need to keep that exclusion for now. I'll move it to `.rubocop_todo.yml` though so it's flagged as something that needs attention when we have more time.